### PR TITLE
Cm notebook

### PR DESCRIPTION
**Pull request by GabrieleMeoni**

### CODE REVIEW

**Comment by GabrieleMeoni** [comment]
[Add a confusion matrix](https://trello.com/c/NngV9dwP/326-add-a-confusion-matrix)

File: datasets/eurosat_rgb_dataset.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 class EurosatRGBDataset(torch.utils.data.Dataset):
     """EurosatRGB dataset"""
 
-    def __init__(self, train, root_dir="data/EuroSAT_RGB/", transform=None, seed=42):
+    def __init__(self, train, root_dir="/home/gabrielemeoni/project/SSLRS/data/EuroSAT_RGB/", transform=None, seed=42):
```
Comment (gomezzz) [inline datasets/eurosat_rgb_dataset.py:16]: pls revert this

File: datasets/ucm_dataset.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ class UCMDataset(torch.utils.data.Dataset):
     """UCM dataset."""
 
     def __init__(
-        self, train, root_dir="data/UCMerced_LandUse/Images/", transform=None, seed=42, resize_to=(224, 224)
+        self, train, root_dir="/home/gabrielemeoni/project/SSLRS/data/UCMerced_LandUse/Images/", transform=None, seed=42, resize_to=(224, 224)
```
Comment (gomezzz) [inline datasets/ucm_dataset.py:17]: pls revert this

File: utils.py [hunk 3]
```diff
@@ -10,12 +10,25 @@
 import seaborn as sn
 import matplotlib.pyplot as plt
 
-def plot_cmatrix(pred,labels,encoding, figsize=(8, 5),dpi=150, font_scale=1.2, save_fig_name=None):
-    cm = confusion_matrix(labels,pred)
-    sn.set(font_scale=font_scale) # for label size
+def plot_cmatrix(preds,labels,encoding, figsize=(8, 5),dpi=150, class_names_font_scale=1.2, matrix_font_size=12, save_fig_name=None):
```
Comment (gomezzz) [inline utils.py:13]: could you add a docstring for this method?

**Review by gomezzz — CHANGES_REQUESTED** [review]

**Review by gomezzz — APPROVED** [review]

**Pull request merged**